gosuper/gosuper/api.go

```diff
@@ -124,23 +125,37 @@ func PurgeHandler(writer http.ResponseWriter, request *http.Request) {
 
 	if appId, err := parsePurgeBody(request); err != nil {
 		sendBadRequest("Invalid request")
-	} else if appId == "" {
+		return
+	}
+	if appId == "" {
```

Comment:
This and other cases will fail due to `appId` not being available in the scope. A variable declared like `foo` in:

```go
if foo := bar; ... {
}
```

Are in scope only for the if block AND any else blocks. This is why the `if`/`else if` was used here to limit the scope of these variables.

Your code here will not compile because of `appId` not being declared.